backend.native/tests/build.gradle

```diff
@@ -240,6 +240,7 @@ run {
     dependsOn(tasksOf(KonanTest))
     // Add framework tests
     dependsOn(tasksOf(FrameworkTest))
+    dependsOn(tasksOf(MetadataComparisonTest))
```

Comment:
We might need to add this test to sanity.
Also there are probably other test kinds that don't run with `run` or `sanity`. For example, `CoverageTest`.

Reply:
I'm looking through all our tests to find out what else isn't run in a `run {}` and in `sanity {}` but could have been.